Students/Benny/3. Protocols and bridging/Questions.md

```diff
@@ -13,7 +13,7 @@
 
 1. What's the importance of 'arrayWithCapacity' when mapping an array. Is this required?
```

Comment:
The main difference performance. ArrayWithCapcity reserves memmory where the mapped array will go into. If you do not do this code will work but with every added element the processor will have to go look for more memory for the next elements. This is more processor intensive the reserve the memory up front if you know the expected length of the array before you calculated the data.